src/component/axis/Axis.jsx

```diff
@@ -205,15 +201,21 @@ export class AxisRenderable extends React.PureComponent {
             0
         );
 
-        return <line
-            x1={x1}
-            y1={y1}
-            x2={x2}
-            y2={y2}
-            stroke='black'
-            strokeWidth={strokeWidth}
-            {...this.props.lineProps}
-        />;
+        const axisLineProps = {
+            x: x1,
+            y: y1,
+            length: this.props[this.getLengthProp(position)],
```

Comment:
This needs position, or for length to be replaced with width/height depending on the axis - eg.

```
[this.getLengthProp(position)]: this.props[this.getLengthProp(position)]
```